src/main.rs

```diff
@@ -1,14 +1,30 @@
 mod application;
+mod db;
 mod engine;
 
+use dotenv::dotenv;
 use log::info;
 use std::collections::HashMap;
 
 use crate::application::Application;
+use crate::db::Database;
 use crate::engine::docker_engine::DockerEngine;
 use crate::engine::engine::Engine;
 
 fn main() {
+    // Load env variables
+    dotenv().ok();
+
+    // Setting up the connection to postgres db
+    let postgres_db_user =
+        std::env::var("POMEGRANATE_DB_USER").expect("POMEGRANATE_DB_USER must be set.");
+    let postgres_db_pass =
+        std::env::var("POMEGRANATE_DB_PWD").expect("POMEGRANATE_DB_PWD must be set.");
+    let postgres_db_url =
+        std::env::var("POMEGRANATE_DB_URL").expect("POMEGRANATE_DB_URL must be set.");
+    let postgres_db_port =
+        std::env::var("POMEGRANATE_DB_PORT").expect("POMEGRANATE_DB_PORT must be set.");
+
```

Comment:
We might want to create a `config.rs` file to avoid overloading the `main()` function with config loading.

Reply:
Done